site/_data/config/bk_server.yaml

```diff
@@ -538,6 +538,9 @@ groups:
         - Twitter Ostrich   : org.apache.bookkeeper.stats.twitter.ostrich.OstrichProvider
         - Twitter Science   : org.apache.bookkeeper.stats.twitter.science.TwitterStatsProvider
     default: org.apache.bookkeeper.stats.prometheus.PrometheusMetricsProvider
+  - param: limitStatsLogging
+    description: option to limit stats logging
```

Comment:
Honestly from this description I can't understand what 'stats logging' really means in this context.
It seems that we are disabling stats at all

@sijie @reddycharan